frontend/public/components/nav/items.tsx

```diff
@@ -175,6 +182,8 @@ export type ResourceClusterLinkProps = NavLinkProps & {
 
 export type HrefLinkProps = NavLinkProps & {
   href: string;
+  namespaced: boolean;
```

Comment:
This should be optional. Defaults to `false`.
This would reduce the overall size of this PR since a lot of changes involve setting this value to `false`.
```suggestion
  namespaced?: boolean;
```